dataflows/processors/dumpers/file_formats.py

```diff
@@ -163,7 +163,7 @@ class JSONFormat(FileFormat):
         'geopoint': lambda d: list(map(float, d)),
         'yearmonth': lambda d: '{:04d}-{:02d}'.format(*d),
     }
-    DEFAULT_SERIALIZER = identity
+    DEFAULT_SERIALIZER = str
```

Comment:
Not sure why this is needed - this only affects to fields of types which don't appear in the SERIALIZERS dict - do you have such fields in your data?

Reply:
This is linked to #150

Reply:
This should be resolved now (use version 0.2.0)